improver_tests/acceptance/SHA256SUMS

```diff
@@ -795,7 +795,7 @@ c44a00d49912f13fb17256fa0e01d9425977b5c94f2d41c602c657d39deb74cc  ./weather-symb
 90a6495f65e053e943dd4dcc9731da274f11bc19a3817a70c057218ed8eaa4de  ./weather-symbol-modes/spot_input/20201209T1600Z-weather_symbols-PT01H.nc
 a981f6238828b63d7a9f7c597dd234dcc86dd35e2e3c4ba6c3b6feb491ec872e  ./weather-symbol-modes/spot_input/20201209T1700Z-weather_symbols-PT01H.nc
 de4a49bca4cd930328942cf6bb9a1bc1bdd2589120a57a2335ee4ef2449dbd5d  ./weather-symbol-modes/spot_input/20201209T1800Z-weather_symbols-PT01H.nc
-fb999f0d50c13d166527ab54932b9e8567e0d75cd8056981c746f08a1edd4ada  ./weather-symbol-modes/spot_input/kgo.nc
+16d57e10b3c6cb8bddc9048c1ab09e11503cecdbe569aef746e95c8cbc23ded1  ./weather-symbol-modes/spot_input/kgo.nc
```

Comment:
I cannot get these to match up with the acceptance test data repo despite resetting both with `--hard` to match your versions. Is it just me?

Reply:
I hadn't pushed up the latest edits to https://github.com/metoppv/improver_test_data/pull/54, so I think that's why there was a mismatch. All tests pass for me using https://github.com/metoppv/improver_test_data/pull/54.